tekton/images/test-runner/Dockerfile

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GO_VERSION=1.19
+ARG GO_VERSION=1.20
 # Build kubetest independently of the rest
-FROM golang:${GO_VERSION}@sha256:800d58363785632e930999c28faee3cf1f8ff7a86ae3eab54af8f63b365fb915 as kubetest
+FROM golang:${GO_VERSION}@sha256:7e3f47de2f547717ebca35255d2b1f4985962d8ebe85bb70746f586e9dc0ab7b as kubetest
```

Comment:
This is the amd64 digest, but I think that's fine for now, we don't run this on other nodes than x86_64.